admin/Makefile

```diff
@@ -48,3 +52,22 @@ ${BUILD_DIR}/requirements-dev.timestamp: ${BUILD_DIR}/venv.timestamp setup.py
 	#Install the project in editable mode with its testing requirements.
 	${VENV}/bin/pip install -e ".[testing]"
 	touch $@
+
+.PHONY: build_admin
+build_admin:
+	cd .. ; \
```

Comment:
why is it needed?

Reply:
because the commons module has to be shipped into the context sent to the docker daemon to build the image.

Reply:
:-/ or extends this image https://hub.docker.com/r/camptocamp/geomapfish-commons/

Reply:
Oh, I did not know it even existed...

Reply:
It's currently manually built & pushed I guess.

How do you plan to build these docker images ? Jenkins CI ? 
Since there will depend on each other, a tagging system has to be adopted. Or we might just publish `latest` images at the moment...

Reply:
It's built on Travis, and we use the tag 2.3 :-)

Reply:
It is build from c2cgeoportal/commons, why do not building it locally, just before c2cgeoportal_admin ?
@sbrunner : I do not understand why we need c2cgeoportal front office deps in admin

Reply:
c2cgeoportal front office deps ... what do you speak about?
That https://github.com/camptocamp/c2cgeoportal/blob/master/docker/build/requirements.txt ?
If yes to avoid to create too many docker images => save download and disk usage (for clients, not for dev)